src/modules/identityId.ts

```diff
@@ -0,0 +1,21 @@
+import { AuthAPI, AuthIdentityPayload } from './auth-api'
+import { AuthIdentity } from 'decentraland-crypto-fetch'
+
+export async function getIdentityId(identity: AuthIdentity): Promise<string> {
+  if (!identity) {
+    throw new Error('Identity is required')
+  }
+
+  if (!identity.authChain || !identity.ephemeralIdentity) {
```

Comment:
The `authChain` and `ephemeralIdentity` are not undefined by how `AuthIdentity` is typed, would you mind removing these checks?